bin/mrm.js

```diff
@@ -67,10 +69,14 @@ if (isDefaultPreset) {
 We’ve tried to load “mrm-preset-${preset}” and “${preset}” globally installed npm packages.`);
 		process.exit(1);
 	}
-	directories = [path.dirname(presetPath)];
+	const presetDir = path.dirname(presetPath);
+	const presetConfig = getConfigFromCosmiconfig(presetDir, {
+		searchPlaces: ['config.json']
+	});
+	directories.push(presetDir);
+	options = Object.assign({}, presetConfig, options);
```

Comment:
It's not clearly code, but useful for shareable configs via presets, like aliases or company based configs.

Reply:
I'm not sure what this code is doing, could you please add comment?

Reply:
Shareable configs via preset useful for teams, sample `<preset>/config.json`:
```json
{
	"emailValidator": "([\\w\\.]+)@corp\\.mail\\.ru",
	"any-shareable-config": "some-value",
	"aliases": {
		"component": ["license", "readme", "editorconfig", "gitignore", "husky"]
	}
}
```

as default config for task, that can rewrite by developer config, if needed.

Reply:
Could you add some docs for this use case?